protected/humhub/modules/admin/views/setting/statistic.php

```diff
@@ -17,7 +19,7 @@
 <?= $form->errorSummary($model); ?>
 
 <div class="form-group">
-    <?= $form->field($model, 'trackingHtmlCode')->widget(CodeMirrorInputWidget::className()); ?>
+    <?= $form->field($model, 'trackingHtmlCode')->widget(CodeMirrorInputWidget::class); ?>
```

Comment:
We should include the below javascript in the InputWidget class so that it doesn't have to be added separately for each view.